thingif/src/main/java/com/kii/thingif/query/HistoryStatesQuery.java

```diff
@@ -2,15 +2,18 @@
 
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
+import android.text.TextUtils;
 
+import com.google.gson.annotations.SerializedName;
 import com.kii.thingif.clause.query.QueryClause;
 
 public class HistoryStatesQuery {
 
-    private @NonNull String alias;
-    private @NonNull QueryClause clause;
+    private transient @NonNull String alias;
+    private transient @NonNull QueryClause clause;
```

Comment:
- `alias` is not need in request body, it is in request url path. 
- `clause` is different in request body.  https://github.com/KiiPlatform/thing-if-AndroidSDK/pull/259/files#diff-12e2b8b1411bb1a56fce72fd0a2187a8R2062